src/scss/components/file/_index.scss

```diff
@@ -0,0 +1,5 @@
+@import "components/form/config";
+@import "components/box/config";
```

Comment:
move these imports into `_config.scss`

Reply:
Import statements moved to `_config.scss`.